File: R/ASCII_30_Del.r
ASCII_30_Del <- function(dectext_ascii)
{
  dectext_ascii[dectext_ascii == 123] <- 228
  dectext_ascii[dectext_ascii == 124] <- 246
  dectext_ascii[dectext_ascii == 125] <- 252
  dectext_ascii[dectext_ascii == 126] <- 223
  
  return(dectext_ascii)
}

File: JavaScript/ASCII_30_Del.js
export function ASCII_30_Del(num) {
    // Add umlaut characters
    if (num === 123)
        num = 228;
    else if (num === 124)
        num = 228;
    else if (num === 125)
        num = 246;
    else if (num === 126)
        num = 252;
    return num
};
